BridgeSDK/BridgeAPI/SBBActivityManager.m

```diff
@@ -138,7 +140,8 @@ - (NSURLSessionDataTask *)getScheduledActivitiesForDaysAhead:(NSInteger)daysAhea
         savedTasks = [self filterTasks:tasks.items forDaysAhead:kMaxAdvance andDaysBehind:kDaysToCache excludeStillValid:YES];
     }
 
-    return [self.networkManager get:kSBBActivityAPI headers:headers parameters:@{@"daysAhead": @(daysAhead), @"offset": [[NSDate date] ISO8601OffsetString]} completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
+    // always request the maximum days ahead from the server so we have them cached
+    return [self.networkManager get:kSBBActivityAPI headers:headers parameters:@{@"daysAhead": @(kMaxAdvance), @"offset": [[NSDate date] ISO8601OffsetString]} completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
```

Comment:
How will caching work for when the schedules change (b/c of changes to data groups)? This still does a replacement, right? (i.e. server wins)